source/agora/node/FullNode.d

```diff
@@ -181,7 +174,7 @@ public class Node : API
 
             bool isNominating ()
             {
-                return this.config.node.is_validator &&
+                return (this.nominator !is null) &&
```

Comment:
A FullNode is not a Validator, right? Therefore it will never have a Nominator instance. So what is the purpose of this check here if it will never return true?

One option is to replace it with an inline delegate like so:

```D
// false: full node is never in the middle of nominating
this.network.startPeriodicCatchup(this.ledger, () => false);
```

But a better solution is to change `startPeriodicCatchup` to make the delegate optional.